spacy_llm/tasks/ner.py

```diff
@@ -166,21 +168,26 @@ def initialize(
             for initialization.
         nlp (Language): Language instance.
         labels (List[str]): Optional list of labels.
+        infer_prompt_examples (int): How many prompt examples to infer from the Example objects. 0 by default.
```

Comment:
I suggest to make this an `int` instead of the `bool` we had before, so users can give a larger file with examples (to infer labels from) but only have like 2 in the prompt.

Reply:
And to have any negative number (e.g. `-1` signals "all examples")